bundle/manifests/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -109,7 +109,7 @@ metadata:
     capabilities: Basic Install
     categories: Integration & Delivery
     containerImage: quay.io/kuadrant/kuadrant-operator:latest
-    createdAt: "2024-12-09T09:56:27Z"
+    createdAt: "2025-02-10T11:02:45Z"
```

Comment:
I expect kuadrant operator new required permission to lie here. Try running `make bundle` and see if this file is changed.